x-pack/plugins/synthetics/public/legacy_uptime/components/fleet_package/synthetics_edit_policy_extension_wrapper.test.tsx

```diff
@@ -0,0 +1,368 @@
+              type: 'text',
+            },
+            name: {
+              type: 'text',
+            },
+            schedule: {
+              value: '"@every 5s"',
+              type: 'text',
+            },
+            wait: {
+              value: '1s',
+              type: 'text',
+            },
+            hosts: {
+              type: 'text',
+            },
+            'service.name': {
+              type: 'text',
+            },
+            timeout: {
+              type: 'text',
+            },
+            max_redirects: {
+              type: 'integer',
+            },
+            tags: {
+              type: 'yaml',
+            },
+          },
+        },
+      ],
+    },
+    {
+      type: 'synthetics/browser',
+      enabled: false,
+      streams: [
+        {
+          enabled: false,
+          data_stream: {
+            type: 'synthetics',
+            dataset: 'browser',
+          },
+          vars: {
+            type: {
+              value: 'browser',
+              type: 'text',
+            },
+            name: {
+              value: 'Sample name',
+              type: 'text',
+            },
+            schedule: {
+              value: '"@every 5s"',
+              type: 'text',
+            },
+            'source.zip_url.url': {
+              type: 'text',
+            },
+            'source.zip_url.username': {
+              type: 'text',
+            },
+            'source.zip_url.password': {
+              type: 'password',
+            },
+            'source.zip_url.folder': {
+              type: 'text',
+            },
+            'source.inline.script': {
+              type: 'yaml',
+            },
+            timeout: {
+              type: 'text',
+            },
+            tags: {
+              type: 'yaml',
+            },
+          },
+        },
+      ],
+    },
+  ],
+  package: {
+    name: 'synthetics',
+    title: 'Elastic Synthetics',
+    version: '0.66.0',
+  },
+};
+
+const defaultCurrentPolicy: any = {
+  ...defaultNewPolicy,
+  id: '',
+  revision: '',
+  updated_at: '',
+  updated_by: '',
+  created_at: '',
+  created_by: '',
+};
+
+const defaultHTTPConfig = defaultConfig[DataStream.HTTP];
+const defaultICMPConfig = defaultConfig[DataStream.ICMP];
+const defaultTCPConfig = defaultConfig[DataStream.TCP];
+const defaultBrowserConfig = defaultConfig[DataStream.BROWSER];
+
+describe('<SyntheticsPolicyEditExtension />', () => {
+  const onChange = jest.fn();
+  const WrappedComponent = ({ policy = defaultCurrentPolicy, newPolicy = defaultNewPolicy }) => {
+    return (
+      <SyntheticsPolicyEditExtensionWrapper
+        policy={policy}
+        newPolicy={newPolicy}
+        onChange={onChange}
+      />
+    );
+  };
+
+  it('shows deprecation notice', async () => {
+    const { getByText } = render(<WrappedComponent />);
+
+    expect(
+      getByText('Synthetic Monitoring is now available out of the box in Synthetics')
+    ).toBeInTheDocument();
+  });
```

Comment:
might be worth adding a unit test for where it shows edit link for managed integeration policies added via monitor management.

Reply:
I thought about that, but it's also covered by e2e tests. Would be simple enough to add a unit test as well, thought.